src/esp/scene/GibsonSemanticScene.cpp

```diff
@@ -69,12 +69,12 @@ bool SemanticScene::buildGibsonHouse(const io::JsonDocument& jsonDoc,
       object->category_ = std::move(category);
     }
 
-    const auto& jsonCenter = jsonObject["location"];
-    if (!jsonCenter.IsNull()) {
+    if (jsonObject.HasMember("location")) {
+      const auto& jsonCenter = jsonObject["location"];
```

Comment:
Hm... but `HasMember()` forces a double lookup, which isn't great. `FindMember()` might be better in both cases?

Reply:
Cool! Made the changes here for this PR. Have another PR pending making the changes in the rest of the codebase.